clang/test/lab1/vyunov_danila/always_inline_attr.cpp

```diff
@@ -0,0 +1,135 @@
+// RUN: %clang_cc1 -load %llvmshlibdir/AlwaysInlinePlugin%pluginext -plugin AddAlwaysInline %s 1>&1 | FileCheck %s --check-prefix=SUM
```

Comment:
Could you please check presence of the attribute in AST tree?

Reply:
Can I solve this with regular expressions? Or is there an easier way?

Reply:
You should pass `ast-dump` option into a test command to print out AST tree and make checks in that tree instead of the plugin log.

Reply:
Thanks! But doesn't ast-dump show the tree before the change?

Reply:
I guess it should look something like this. Is this the solution to your remark?